server/store/datastore/migration/026_convert_to_new_pipeline_errors_format.go

```diff
@@ -59,25 +64,36 @@ var convertToNewPipelineErrorFormat = task{
 			return err
```

Comment:
Okay error is actually coming from using `.Sync` twice in the migration / session.